srcCpp/PerftestTransport.h

```diff
@@ -11,6 +11,11 @@
 #include <sstream>
 #include "ParameterManager.h"
 
+
+#define DEFAULT_MESSAGE_SIZE_MAX 65536
+#define MESSAGE_SIZE_MAX_NOT_SET -1
+#define MESSAGE_OVERHEAD_BYTES 2536
```

Comment:
@fercsrti77 We choose this value so in the default case the change to asynchronous is done in 63000, this is quite random though. What value should we use here?

Reply:
Instead use this:

include/share/commend/commend_writerService.h:#define COMMEND_WRITER_MAX_RTPS_OVERHEAD